### ISSUE TITLE
[A11y] Explore Collection Landing Page

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3256]
### Operating System

MacOS

### Browser

Safari

### Affected page

Explore Collections

### Admin user?

No

### Issue Description

Link: https://oregondigital.org/collections/abigail-scott-duniway-coll232b?locale=en

When using VoiceOver on a Mac. 

The read order for the collection is not in proper order. When it reads the collection description it then skips "Read more" and "Additional Colletion Information" and instead reads the institution logo.

### Suggested Fix

When reading the collection it should read in the following order:

Collection Title
Collection Description
Read More
Additional Collection Information
Institution Logo

@jsimic should institution logo be read first or is this order fine?

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

The followinging items should be read together:

Collection Title
Collection Description
Read More
Additional Collection Information
